src/transport/MessageCounter.h

```diff
@@ -106,8 +107,12 @@ class GlobalEncryptedMessageCounter : public MessageCounter
 class LocalSessionMessageCounter : public MessageCounter
 {
 public:
-    static constexpr uint32_t kInitialValue = 1;
-    LocalSessionMessageCounter() : value(kInitialValue) {}
+    static constexpr uint32_t kInitialValue = 1; ///< Used for initializing peer counter
+
+    /**
+     * Initialize a local message counter with random value between [0, 2^28-1].
```

Comment:
Are you referring to [this spec](https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/secure_channel/Message_Format.adoc#111-message-counter-initialization)? If that's the case it should be nice to include some context.
```suggestion
     * Initialize a local message counter with random value between [0, 2^28-1]. This increases the difficulty of traffic analysis attacks by making it harder to determine how long a particular session has been open.
```